base/strings/string.jl

```diff
@@ -425,6 +425,17 @@ function reverse(s::String)
     String(buf)
 end
 
+"""
+    repeat(s::String, r::Integer)
+
+Repeat a string `r` times.
```

Comment:
little odd that repeat is currently in the linalg doc index, maybe should be moved to more general collections section